book/website/project-design/logical-frameworks-overview/theory-of-change.md

```diff
@@ -0,0 +1,45 @@
+
+(pd-theory-of-change)=
+
+## Theory of Change (ToC)
+
+A theory of change helps you connect the dots between your actions and the positive change you want to see in the world. It's a powerful tool for planning and evaluating the effectiveness of your efforts, whether you're working on social causes, personal goals, or any other project. It encourages you to think critically about the steps needed to make a real impact.
+
+- **Definition**: Originating from theory-driven evaluation in the 1990s, ToC is an extensive methodology that illustrates complex pathways leading to a desired change or impact.
+- **Nature**: Explanatory, presenting the comprehensive picture with all potential pathways to change.
+- **Structure**: Adaptable, can be a flow chart or an intricate diagram with narrative text. Includes cyclical processes, feedback loops, and multiple pathways.
+- **Focus**: Addresses broader aspects of change, including uncontrollable factors and the wider context of culture and power relations.
+- **Process**: Involves deep critical reflection, logical thinking, and extensive data. It starts by identifying the goal and works backward, asking "if we do X, then Y will happen because…".
+- **Usage**: Ideal for complex projects across multiple sectors, requiring rigorous planning and a thorough understanding of the intervention's impact.
+
+<img width="603" alt="Screenshot 2023-11-17 at 09 13 35" src="https://github.com/the-turing-way/the-turing-way/assets/53487593/380d3c0d-4c1a-4965-bba0-c4a16487437e">
+
+
+## Logical Framework (LogFrame)
+
+A Logical Framework is like zooming in on the specific pathway that your program deals with and creating a neat, orderly structure for it. This makes it easier for you and the donor to monitor program implementation.
```

Comment:
```suggestion
## Logical Framework

A Logical Framework (Log Frame) is like zooming in on the specific pathway that your program deals with and creating a neat, orderly structure for it. This makes it easier for you and the donor to monitor program implementation.
```

This portmanteau also needs to be introduced in the logical-frameworks-overview.md file where it is used as well. See earlier confused comment :)